src/button/tests/functional/Button.ts

```diff
@@ -0,0 +1,73 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=button')
+		.setFindTimeout(5000);
+}
+
+registerSuite({
+	name: 'Button',
+
+	'button should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('button:first-of-type')
+		// `isDisplayed` is broken in Safari 10, commented out for now. See https://github.com/SeleniumHQ/selenium/issues/3029
```

Comment:
Dylan suggested a slightly different approach: https://github.com/dojo/widgets/pull/168#discussion_r116070666, so that's why the comment is there. I've removed the commented code, however I do have a concern that it might get lost over time once the issue with Safari is fixed.